firmware/controllers/algo/launch_control.cpp

```diff
@@ -24,7 +24,11 @@
 #include "engine_state.h"
 #include "advance_map.h"
 
+static bool isInit = false;
 static Logging *logger;
+static efitick_t launchTimer = 0;
```

Comment:
move this state inside the launch control instance, right now it's not testable!